### DISCUSSION TITLE
Sequential screening

### DISCUSSION THREAD

**Discussion by Jetvdg** [asreview/asreview#1772] [Q&A] [answered]
Hi all,
I have shared my ASReview project with my colleague so he could continue the screening process while I was absent. A couple of days ago we saw that when I open ASReview, I still see my project as how I left it (in terms of the numbers of screened abstracts), although my colleague has screened about 150 abstracts. Also in the history I cannot see the abstracts he has screened. We both use the ASReview version 1.6.2. Does someone know how we can synchronise the project, so the sequential screening will work out? Thank you in advance!

**Comment by Rensvandeschoot** [comment, marked as answer]
For clarification: Did you export the project file and share it with your colleague and did your colleague then import the project file and continue with the screening? If so, are the screened records missing when you open the project file of your colleague or your own project file
 on your system?

**Reply by Jetvdg** [reply]
Goodmorning,
We have found the problem, it was indeed that we should have imported the new version. Thank you for your fast reply!
